drizzlepac/hapsequencer.py

```diff
@@ -1035,6 +1037,8 @@ def update_wcs_in_visit(tdp):
         log.info("The active WCS solution for each Grism/Prism and direct image is not changed.")
 
     log.info("Grism product list: {}".format(grism_product_list))
+    # MDD
+    sys.exit(0)
```

Comment:
Did you mean to exit before returning the list?

Reply:
Sorry, code was still "In Progress" and being debugged.  The lines 1040/1041 have been removed.